aws/routeAwsGet.go

```diff
@@ -18,21 +18,73 @@ import (
 	"database/sql"
 	"errors"
 	"net/http"
+	"strconv"
+	"strings"
 
 	"github.com/trackit/jsonlog"
 
 	"github.com/trackit/trackit2/db"
+	"github.com/trackit/trackit2/models"
 	"github.com/trackit/trackit2/routes"
 	"github.com/trackit/trackit2/users"
 )
 
+// AwsAccountsFromUserIDByAccountID retrieves rows from 'trackit.aws_account' as AwsAccount.
+// The result is filtered by a slice of accountID
+func AwsAccountsFromUserIDByAccountID(db models.XODB, userID int, accountIDs []int) ([]AwsAccount, error) {
```

Comment:
Ideally this should be done with XO templates. Let’s leave this here and I’ll create a task to track this.